src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/config/MessagingMultiNodeTestBase.kt

```diff
@@ -117,6 +118,23 @@ abstract class MessagingMultiNodeTestBase : IntegrationTestBase() {
     return response!!.messageId()
   }
 
+  fun publishProbationEvent(eventType: String, probationEvent: ProbationEvent): String {
+    val probationEventAsString = objectMapper.writeValueAsString(probationEvent)
+    val publishRequest = PublishRequest.builder().topicArn(domainEventsTopic?.arn)
+      .message(probationEventAsString)
+      .messageAttributes(
+        mapOf(
+          "eventType" to MessageAttributeValue.builder().dataType("String")
+            .stringValue(eventType).build(),
+        ),
+      ).build()
+
+    val response = domainEventsTopic?.snsClient?.publish(publishRequest)?.get()
+
+    expectNoMessagesOn(probationEventsQueue)
+    return response!!.messageId()
+  }
+
```

Comment:
this has a lot in common with the `publishDomainEvent` method above - looks like we could reuse much of that method